app/src/test/java/org/pesmypetcare/mypetcare/controllers/user/TestTrSendFirebaseMessagingToken.java

```diff
@@ -0,0 +1,40 @@
+package org.pesmypetcare.mypetcare.controllers.user;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.pesmypetcare.mypetcare.controllers.user.EmptyMessagingTokenException;
+import org.pesmypetcare.mypetcare.controllers.user.TrSendFirebaseMessagingToken;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [No need to import a type that lives in the same package](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5561369)